app/assets/javascripts/api/saved_jobs.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```

Comment:
I realize that I have been generating controllers and models so there may be some unused files. I will go through and delete those